firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/schema.ts

```diff
@@ -70,7 +70,8 @@ export const documentIdField = {
  * We cannot specify a schema for view creation, and all view columns default
  * to the NULLABLE mode.
  */
-export const RawChangelogViewSchema: any = {
+
+export const RawChangelogViewSchema = {
```

Comment:
Types have been removed, can we add these back in?

Reply:
Typescript will automatically assign Types if none are provided. I think it is a better solution than `any` as it is checking them now.